Uppsala/UppEt1.xml

```diff
@@ -0,0 +1,470 @@
+                                
+                                <item xml:id="a8">
+                                    <locus target="#230v"/>
+                                    <desc>Armenian note, identified by 
+                                        <bibl>
+                                            <ptr target="bm:Zettersteen1899Upsala"/>
+                                            <citedRange unit="page">509</citedRange>
+                                        </bibl>
+                                        as a name.</desc>
+                                </item>
+                                
+                                <item xml:id="a9">
+                                    <locus target="#230v #231r"/>
+                                    <desc>Armenian seal stamps in red ink.</desc>
+                                </item>
+                                
+                                <item xml:id="a10">
+                                    <locus target="#272ra"/>
+                                    <desc>The Armenian alphabet.</desc>
+                                </item>
+                            </list>
+                        </additions>
+                        
+                        <bindingDesc>
+                            <binding xml:id="binding">
+                                <decoNote xml:id="b1" type="Boards">Wooden boards covered with undecorated leather. According to
+                                    <bibl><ptr target="bm:Loefgren1974Katalog"/><citedRange unit="page">35</citedRange></bibl>,
+                                    the binding is probably not Ethiopian (<foreign xml:lang="de">kaum äthiopisch</foreign>),
+                                    whereas
+                                    <bibl>
+                                        <ptr target="bm:Zettersteen1899Upsala"/>
+                                        <citedRange unit="page">509</citedRange>
+                                    </bibl> states that it is Ethiopian.</decoNote>
+                            </binding>
+                        </bindingDesc>
+                        
+                    </physDesc>
+                                        
+                    <history>
+                        <origin>
+                            <origDate notBefore="1500" notAfter="1599">16th century</origDate>
+                        </origin>
+                        <provenance>
+                            <bibl>
+                                The ultimate origin of the manuscript is unknown. It made its earliest appearance in
+                                <ref type="place" corresp="LOC2085Cairo"/> in <date when="1664">1664</date>,
+                                when it served as a partial <foreign xml:lang="de">Vorlage</foreign> for 
+                                <ref type="mss" corresp="BNCFcod1"/>, a manuscript scribed by <ref type="pers" corresp="PRS10036Wansleb"/>
```

Comment:
maybe copied instead of scribed would be better, because it is more common?

Reply:
I think "copied" would make it ambiguous in this context :(